lib/ddtrace/monkey.rb

```diff
@@ -0,0 +1,14 @@
+module Datadog
+  # Monkey is used for monkey-patching 3rd party libs.
+  class Monkey
+    def self.patch_all
+      require 'ddtrace/contrib/elasticsearch/core' if \
```

Comment:
Wouldn't do either, but we happen to have a paranoid CI that runs Rubocop and won't bargain. Kind of weird, end up as a "there's only one way to do it" world. OTOH, it traps a good deal of stupid mistakes.